.gitignore

```diff
@@ -1,15 +1,261 @@
-# build and binary files
```

Comment:
Same here. Shortly after I started this project, I was a node developer for many years, hence why I'm catching up on some .NET goodness. I used atom for a long time, until I found VSCode was actually better and faster (even in Mac). Thanks!